src/02-data-types.js

```diff
@@ -79,7 +89,9 @@ function convertToWhisperText(text) {}
  *  checkIfCharacterIsInString("hello there", "a");
  *  //> false
  */
-function checkIfCharacterIsInString(text, character) {}
+function checkIfCharacterIsInString(text, character) {
+  return text.includes(character)
```

Comment:
Try to solve this without `includes()` it is a good challenge and will help understand loops.